packages/nvidia_nat_core/src/nat/front_ends/fastapi/message_handler.py

```
@@ -408,6 +425,13 @@ async def _run_workflow(self,
 
                     await self.create_websocket_message(data_model=value, status=WebSocketMessageStatus.IN_PROGRESS)
 
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)